src/js/components/Button/Button.js

```diff
@@ -399,6 +433,31 @@ const Button = forwardRef(
       contents = <Badge content={badgeProp}>{contents}</Badge>;
     }
 
+    if (busy || success) {
+      // match what the label will use
+      let animationColor;
+      if (kind) {
+        if (!plain) {
+          animationColor =
+            (hover && getIconColor(themePaths.hover, theme)) ||
+            getIconColor(themePaths.base, theme, color, kind);
+        }
+      } else if (primary) {
+        animationColor =
+          theme.global.colors.text[isDarkBackground() ? 'dark' : 'light'];
+      }
+
+      contents = (
+        <Box style={{ position: 'relative' }}>
```

Comment:
Add comment about why style tag and position is necessary

Reply:
added comment